tests/cases/fourslash/getOccurrencesConst04.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference path='fourslash.ts' />
 
 ////export const class C {
-////    private static c/*1*/onst foo;
+////    private static const f/*1*/oo;

```

Comment:
The fix shouldn't be to change the marker position, it's to ensure that the occurrence count is 0.